hypothesis-python/src/hypothesis/extra/pytestplugin.py

```diff
@@ -213,3 +201,22 @@ def note_strategy_is_not_test(*args, **kwargs):
 
 def load():
     pass
+
+
+if LooseVersion(pytest.__version__) < "4.3":  # pragma: no cover
+    from warnings import warn
+    from hypothesis.errors import HypothesisWarning
+
+    warn(
+        """
+    You are using Pytest version %s.  Hypothesis tests work with any test
+    runner, but our Pytest plugin requires Pytest  4.3 or newer.
+    Note that the Pytest developers no longer support this version either!
+    Disabling the Hypothesis pytest plugin..."""
+        % (pytest.__version__),
+        HypothesisWarning,
+    )
+    g = globals()
```

Comment:
Does this work? I'm very suspicious of this code.

Could we instead just wrap the hole module in an if statement? I know it makes the diff larger but it's less mysterious and likely more robust.